native-image/graalvm-native-image-quick-start/README.md

```diff
@@ -147,25 +147,29 @@ executable that does not require a JDK to run, is fast to start and efficient.
 
    ![](images/RMIL_Technology_Laptop_Bark_RGB_50.png#input)
    ```bash
-    native-image -jar ./target/graalvmnidemos-1.0-SNAPSHOT-jar-with-dependencies.jar --no-fallback -H:Class=oracle.App -H:Name=file-count
-    ```
+   native-image \
+     -jar ./target/graalvmnidemos-1.0-SNAPSHOT-jar-with-dependencies.jar \
+     --no-fallback \
+     -H:Class=oracle.App \
+     -H:Name=file-count
```

Comment:
Added newlines to separate the commands, as they're referred to separately below.